packages/passport-interface/src/SubscriptionManager.ts

```diff
@@ -91,11 +100,15 @@ export class FeedSubscriptionManager {
    * Returns the successful responses. Failures will be recorded in
    * `this.errors` for display to the user.
    */
-  public async pollSubscriptions(): Promise<SubscriptionActions[]> {
+  public async pollSubscriptions(
+    identity: Identity
```

Comment:
I don't think this makes sense - subscriptions shouldn't really know about the notion of an identity - to the subscription manager, credentials and pcds it gets back from a feed should be transparent.

Reply:
But subscription manager is responsible for polling feeds, and it can't poll a feed without a credential, and a credential must be a SemaphoreSignaturePCD which contains a timestamp, and these have to be freshly created, which requires the identity.

Reply:
Subscription manager should be responsible for polling feeds, but I don't think it should be responsible for knowing how to generate all possible credentials. That should be the job of something else.